specs/trip_dispatch_spec.rb

```diff
@@ -89,4 +89,19 @@
       passenger.trips.must_include trip
     end
   end
+
+  describe "adds new trips" do
+    before do
+      @dispatcher = RideShare::TripDispatcher.new
```

Comment:
There are a couple other things you should be testing around the `request_trip` method:

- Was the trip added to the trip lists of the driver and passenger?
- What if there is no available driver?

The second is an important case, and is exactly the sort of situation where software could do something unexpected. You might set up the test as follows:

```ruby
it 'raises an error if no driver is available' do
  @dispatcher.drivers.each do |driver|
    # This may require an attr_writer in Driver
    driver.status = :UNAVAILABLE
  end
  proc {
    @dispatcher.request_trip(2)
  }.must_raise StandardError
end
```